evm_loader/program/Cargo.toml

```diff
@@ -40,20 +40,21 @@ default = ["custom-heap"]
 linked_list_allocator = { version = "0.10", default-features = false }
 evm-loader-macro = { path = "../program-macro" }
 solana-program.workspace = true
-spl-token = { version = "~4.0", default-features = false, features = ["no-entrypoint"] }
-spl-associated-token-account = { version = "~2.3", default-features = false, features = ["no-entrypoint"] }
-mpl-token-metadata = { version = "~4.1", default-features = false }
+spl-token = { version = "~6.0", default-features = false, features = ["no-entrypoint"] }
+spl-associated-token-account.workspace = true
+mpl-token-metadata = { version = "5.1.0", default-features = false }
 thiserror = "1.0"
 arrayref = "0.3.8"
 hex = "0.4.3"
 ripemd = "0.1"
 rlp = "0.5"
 static_assertions = "1"
-borsh = "0.10"
+borsh = "1.5.2"
+borsh010 = { version = "0.10.4", package = "borsh" }
```

Comment:
What is the difference between borsch v1.5.2 and v0.10.4? Why do we need to use both of them?

Reply:
There is no difference for us between this two crates. Mpl-token-metadata crate depends on >=0.9, <1.0 [borsh](https://crates.io/crates/mpl-token-metadata/5.1.0/dependencies). [Solana-program](https://crates.io/crates/solana-program/2.0.15/dependencies) depends on both borsh 1.5 and 0.10. So, we need to tell which version of borsh. Because it does not support deserialization from v1.5 trait.